mp/src/game/shared/momentum/fx_mom_shared.cpp

```diff
@@ -178,16 +180,42 @@ void FX_FireBullets(int iEntIndex, const Vector &vOrigin, const QAngle &vAngles,
         lagcompensation->StartLagCompensation(pPlayer, pPlayer->GetCurrentCommand());
 #endif
 
-    for (int iBullet = 0; iBullet < pWeaponInfo->m_iBullets; iBullet++)
-    {
-        RandomSeed(iSeed); // init random system with this seed
+    // TF2 spread pattern
+    Vector vecFixedPattern[] = {
+        Vector(0, 0, 0),        Vector(1, 0, 0),       Vector(-1, 0, 0),        Vector(0, -1, 0),       Vector(0, 1, 0),
+        Vector(0.85, -0.85, 0), Vector(0.85, 0.85, 0), Vector(-0.85, -0.85, 0), Vector(-0.85, 0.85, 0), Vector(0, 0, 0),
+    };
+
+    int iTotalBullets = pWeaponInfo->m_iBullets;
+    bool bNoSpread = false;
 
-        // Get circular gaussian spread.
+    bNoSpread = iTotalBullets > 1 && mom_fixed_spread.GetBool();
```

Comment:
Join the declaration and assignment. Also using a negative name as a positive boolean can lead to logic problems when reading the code. I suggest:

`bool bFixedSpread = iTotalBullets > 1 && mom_fixed_spread.GetBool()`